package.json

```diff
@@ -7,34 +7,43 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
-    "format": "prettier . --write"
+    "format": "prettier . --write",
+    "prepare": "husky install",
+    "postinstall": "husky install"
```

Comment:
one works with yarn and one works with npm

Reply:
Is the added complexity a necessity? If we support Yarn as well then we'll need to make documentation changes as well which I think shouldn't be priority right now lol. I would really like to get started developing the website ASAP instead of tinkering around with the trivial stuffs.

So, let's stick to `npm` stuff for now.

Reply:
I doubt we need to support `yarn` since we specifically mentioned in the docs to use `npm` instead. So, could you make those changes as well?

Reply:
you cant force any dev/contributors to use npm specifically, this repo is becoming opiniated.